src/lib.rs

```diff
@@ -119,29 +119,23 @@ pub struct BlockBasedOptions {
 /// use rocksdb::{Options, DB};
 /// use rocksdb::DBCompactionStyle;
 ///
-/// fn badly_tuned_for_somebody_elses_disk() -> DB {
-///    let path = "path/for/rocksdb/storageX";
-///    let mut opts = Options::default();
-///    opts.create_if_missing(true);
-///    opts.set_max_open_files(10000);
-///    opts.set_use_fsync(false);
-///    opts.set_bytes_per_sync(8388608);
-///    opts.set_disable_data_sync(false);
-///    opts.optimize_for_point_lookup(1024);
-///    opts.set_table_cache_num_shard_bits(6);
-///    opts.set_max_write_buffer_number(32);
-///    opts.set_write_buffer_size(536870912);
-///    opts.set_target_file_size_base(1073741824);
-///    opts.set_min_write_buffer_number_to_merge(4);
-///    opts.set_level_zero_stop_writes_trigger(2000);
-///    opts.set_level_zero_slowdown_writes_trigger(0);
-///    opts.set_compaction_style(DBCompactionStyle::Universal);
-///    opts.set_max_background_compactions(4);
-///    opts.set_max_background_flushes(4);
-///    opts.set_disable_auto_compactions(true);
-///
-///    DB::open(&opts, path).unwrap()
-/// }
+/// let mut opts = Options::default();
+/// opts.create_if_missing(true);
+/// opts.set_max_open_files(10000);
+/// opts.set_use_fsync(false);
+/// opts.set_bytes_per_sync(8388608);
+/// opts.optimize_for_point_lookup(1024);
+/// opts.set_table_cache_num_shard_bits(6);
+/// opts.set_max_write_buffer_number(32);
+/// opts.set_write_buffer_size(536870912);
+/// opts.set_target_file_size_base(1073741824);
+/// opts.set_min_write_buffer_number_to_merge(4);
+/// opts.set_level_zero_stop_writes_trigger(2000);
+/// opts.set_level_zero_slowdown_writes_trigger(0);
+/// opts.set_compaction_style(DBCompactionStyle::Universal);
+/// opts.set_max_background_compactions(4);
+/// opts.set_max_background_flushes(4);
+/// opts.set_disable_auto_compactions(true);
```

Comment:
Can you leave the part about `badly_tuned_for_somebody_elses_disk`? Far too often, people look up a tunable, and blindly copy what they find in docs somewhere. I would like to encourage people to measure reasonable values for their own workloads and available hardware.